front/src/BottomBar/index.js

```diff
@@ -14,79 +14,119 @@ import pageTypes from "./pageTypes";
  * and the implementation of the functions passed to BottomBar.
  * Author: T.T.P. Franken
  * Author: B. Afonins
+ * Author: G.M. van der Sanden
  *
  * Props:
  * - onSend: function to send a command
  * - changeActivePage: function to change the active page
  * - activePage: a string indicating the current active page
+ * - isConnected: a boolean indicating whether the client is connected to the TRC
  */
 
 const BottomBar = props => {
-  const { changeActivePage, onSend, activePage } = props;
+  const { changeActivePage, onSend, activePage, isConnected } = props;
   const isSettingsPage = activePage === pageTypes.SETTINGS;
   return (
+	<TotalBar className={props.className}>
+	<ConnectedBar id="connectBar" isConnected={props.isConnected}>
+	  {props.isConnected ? "Connected": "Disconnected"}
```

Comment:
connection status text is not centered vertically. use `display: flex` on the parent `ConnectedBar`

Reply:
Did not do this, however, it is centered now. It had to do with some flex-mistakes in the old TotalBar (now Bar), the parent of ConnectionBar.